scanomatic/generics/abstract_model_factory.py

```diff
@@ -241,6 +242,12 @@ def _enforce_model(factory, obj):
                 else:
                     break
 
+        def _get_enum(enum: EnumMeta, name: str) -> Enum:
```

Comment:
why is this needed? loaded models should already have enums as enums when it reaches the factory.

Reply:
The `object_hook` in `jsonizer.py` should have loaded all enums as enums.

Reply:
I'm super confused about this. I've spent I don't know how much time trying to solve this enum parsing issue. But now it just works without it. :confused:
Anyway, I removed it... :facepalm:

Reply:
So it is some times.